src/main/java/com/dev/cinema/dao/OrderDao.java

```diff
@@ -9,7 +9,7 @@
  * @author Sergey Klunniy
  */
 public interface OrderDao {
-    Order create(Order order);
+    Orders create(Orders orders);
```

Comment:
```suggestion
    Order create(Order order);
```
класи повинні називатися в однині

Reply:
+